src/QueueReceiver.Infrastructure/Repositories/ProjectRepository.cs

```diff
@@ -13,9 +13,7 @@ public class ProjectRepository : IProjectRepository
         private readonly ApplicationDbContext _context;
 
         public ProjectRepository(ApplicationDbContext context)
-        {
-            _context = context;
-        }
+            => _context = context;
```

Comment:
Inconsistent usage for context and context.TheDbSet.

PlantRepository and ProjectRepository handles these different